cluster-up/cluster/kind/common.sh

```diff
@@ -154,13 +154,13 @@ function setup_kind() {
 
     echo "ipv6 cni: $IPV6_CNI"
     if [ -z ${IPV6_CNI+x} ]; then
-        echo "no ipv6, safe to install flannel"
-        _kubectl apply -f $KIND_MANIFESTS_DIR/kube-flannel.yaml
+        echo "no ipv6, safe to install calico"
```

Comment:
This change has merit on its own and is not necessarily connected with SR-IOV. Please open a separate PR for it

Reply:
#383